### DISCUSSION TITLE
Toolbar force the button to be in a new line

### DISCUSSION THREAD

**Discussion by michele-cassani** [sibiraj-s/ngx-editor#567] [Q&A]
What I would like to achieve is the ability to force the position of the buttons in the toolbar, I would just need to introduce a placeholder component that forces a "line break". Is there a way already available to achieve this?

**Comment by sibiraj-s** [comment]
Can you be a bit more elaborate? 

For changing the order of the buttons through the configuration here, just provider the options in order you need, the menu items will be rendered in that order. https://sibiraj-s.github.io/ngx-editor/menu/

**Comment by michele-cassani** [comment]
I'll try with a basic example:

If I set the toolbar as
```
toolbar: Toolbar = [
    // default value
    ['bold', 'italic'],
    ['underline', 'strike'],
];
```

the four buttons will be on the same line with just the separator line. What can I do to have on the first line of the toolbar "bold" and "italic" and on a second line toolbar "underline" and "strike"?
